assignments/closure.js

```diff
@@ -1,10 +1,15 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+function subtract(x,y) {
+  return `This is a subtraction closure. Be amazed! The answer is ${x - y}.`
```

Comment:
This isn't a closure example. Let me know if you want to go over closures again.